### ISSUE TITLE
Sensor LYWSD03MMC  multiple Configuration

### ISSUE THREAD

**Issue by guardianbs** [AlexxIT/XiaomiGateway3#632]
Updated from v2.0-alpha.4 to v2.0.1  and the following error is repeated 4 times for EACH ble LYWSD03MMC by Xiaomi on EVERY reboot.

```
2022-02-09 15:22:07 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform xiaomi_gateway3
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 382, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 422, in _async_add_entity
    entity.add_to_platform_start(
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 752, in add_to_platform_start
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: Entity sensor.a4c138f30641_temperature cannot be added a second time to an entity platform
2022-02-09 15:22:07 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform xiaomi_gateway3
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 382, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 422, in _async_add_entity
    entity.add_to_platform_start(
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 752, in add_to_platform_start
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: Entity sensor.a4c138f30641_humidity cannot be added a second time to an entity platform
2022-02-09 15:22:07 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform xiaomi_gateway3
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 382, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 422, in _async_add_entity
    entity.add_to_platform_start(
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 752, in add_to_platform_start
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: Entity sensor.a4c138f30641_battery cannot be added a second time to an entity platform
```

**Comment by AlexxIT** [comment]
Check if v2.0.3 helps

**Comment by pskaua** [comment]
Line 761 on v2.0.3.

`
2022-02-12 21:15:15 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform xiaomi_gateway3
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 382, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 422, in _async_add_entity
    entity.add_to_platform_start(
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 761, in add_to_platform_start
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: Entity sensor.temperature_work_room_e_temperature cannot be added a second time to an entity platform
2022-02-12 21:15:15 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform xiaomi_gateway3
(...)
`

**Comment by AlexxIT** [comment]
How many gateway do you have?

**Comment by pskaua** [comment]
I have two gateways (ZNDMWG03LM) on v1.5.0_0102 firmware.

**Comment by guardianbs** [comment]
I have 3 gateways (ZNDMWG03LM) 2 are on 1.4.7_0115 and 1 is on 1.5.0_0102 firmware.

**Comment by peetereczek** [comment]
LYWSD03MMC sensor is unavailable with integration version v2.0.3 and gateway ZNDMWG03LM firmware v1.5.0_0102. I have one gateway and one LYWSD03MMC sensor only

**Comment by guardianbs** [comment]
I have 10 LYWSD03MMC sensors all visible and working on 3 ZNDMWG03LM gateways (2 firmware v1.4.7_0115 and 1 v1.5.0_0102) with integration  v2.0.3.

**Comment by pskaua** [comment]
Fixed on 2.0.4. :) Thx!

**Comment by guardianbs** [comment]
All good.
